packages/platforms/react-native/android/src/main/java/com/bugsnag/android/performance/ReactNativeSpanContext.java

```diff
@@ -0,0 +1,42 @@
+package com.bugsnag.reactnative.performance;
+
+import androidx.annotation.NonNull;
+import com.bugsnag.android.performance.SpanContext;
+import java.util.UUID;
+import java.util.concurrent.Callable;
+
+class ReactNativeSpanContext implements SpanContext {
+  private final long nativeSpanId;
+  private final UUID nativeTraceId;
+
+  public ReactNativeSpanContext(String spanId, String traceId) {
+    nativeSpanId = Long.parseUnsignedLong(spanId, 16);
+    nativeTraceId = new UUID(
+      Long.parseUnsignedLong(traceId.substring(0, 16), 16),
+      Long.parseUnsignedLong(traceId.substring(16), 16)
+    );
```

Comment:
It feels like `16` for the radix should maybe be made into a constant?
```java
private static final int HEX_RADIX = 16;
```

It might also be worth it to have a `TRACE_ID_MIDPOINT = 16` constant